pkg/chunk/grpc/gRPC.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+package grpc;
+
+service grpc_store {
+    //index-client
+    rpc BatchWrite(WriteBatch) returns (Nothing);
```

Comment:
What's this `Nothing`? Why do we need it?

Reply:
So in proto files. When we have nothing to return back. We maintain an empty message i.e
`message Nothing {};`
If there any workaround to avoid this. I can consider using them.

Reply:
You can use the built-in `google.protobuf.Empty` ([see this](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/empty.proto))